.github/workflows/solutions-uat.yaml

```
@@ -0,0 +1,159 @@
+        uses: helmfile/helmfile-action@v1.9.0
+        with:
+          helmfile-version: 'v0.150.0'
+          helm-plugins: >
+            https://github.com/databus23/helm-diff
+
+      - name: digit deployment
+        run: helmfile -l deployment=pucar,team!=dpg -f digit-helmfile.yaml apply --include-needs=true
+
+        working-directory: deploy-as-code
+        env:
+          HELMFILE_ENV: "solutions-uat"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)